pages/tutorial/index.md

```diff
@@ -17,6 +17,7 @@ header:
 0. [Simulator setup][setup]
 0. [How to write simulations][Simulations]
 0. [The graphical interface][gui]
+0. [YAML for Alchemist][yaml]
```

Comment:
Why is this different than "How to write simulations"?

Reply:
Does this still need to get fixed?

Reply:
I removed the "YAML for Alchemist" entry, now there is no reference to it but the page is still in the repository.
However, I did not change the "How to write simulation" page.
If needed, I can add some examples taken from the tutorials at the end of every section or even link the tutorial repositories.
Moreover, I can join the two pages and, then, remove the not used one.